impress-org/givewp
# Refactor: Update beta feature message component styles

## Description

Updated the styles of the component to match the designs.

## Affects

This affects the styles of the beta feature message component in the admin settings.

## Visuals
Before
<img width="1196" alt="Screenshot 2024-11-01 at 3 11 16 PM" src="https://github.com/user-attachments/assets/7d1a8343-be5e-4640-a21b-7e8f38910921">

After
<img width="1194" alt="Screenshot 2024-11-01 at 3 11 48 PM" src="https://github.com/user-attachments/assets/00bb4ecf-2775-4a08-aa12-413826688872">


## Testing Instructions

1. Install this locally
2. Go to Settings > General > Beta Features
3. Check the updated styles

## Pre-review Checklist
